packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -0,0 +1,281 @@
+ *         {title: "Button 2", onClick: () => {}},
+ *     ]}
+ *     onDismiss={() => {console.log("Has been dismissed.")}}
+ * />
+ * ```
+ */
+const Banner = (props: Props): React.Node => {
+    const {actions, onDismiss, kind, layout = "full-width", text} = props;
+    const layoutstyle = {
+        borderRadius: layout && layout === "full-width" ? 0 : 4,
+    };
+
+    const renderActions = () => {
+        return actions?.filter(Boolean).map((action) => {
+            const handleClick = action.onClick;
+            if (action.href) {
+                return (
+                    <View style={styles.action} key={action.title}>
+                        <Link
+                            kind="primary"
+                            href={action.href}
+                            onClick={handleClick}
+                            aria-label={action.ariaLabel ?? action.title}
+                        >
+                            {action.title}
+                        </Link>
+                    </View>
+                );
+            } else if (action.onClick) {
+                return (
+                    <View style={styles.action} key={action.title}>
+                        <Button
+                            kind="tertiary"
+                            size="medium"
+                            aria-label={action.ariaLabel ?? action.title}
+                            onClick={handleClick}
+                        >
+                            {action.title}
+                        </Button>
+                    </View>
+                );
+            } else {
+                throw new Error(
+                    "An action must have an href or an onClick field.",
+                );
+            }
+        });
+    };
+
+    return (
+        <View
+            style={[
+                styles.containerOuter,
+                layoutstyle,
+                {borderInlineStartColor: colorForKind(kind ?? "info")},
+            ]}
+        >
+            <View
+                style={[
+                    styles.backgroundColor,
+                    layoutstyle,
+                    {backgroundColor: colorForKind(kind ?? "info")},
+                ]}
+            />
+            <View style={styles.containerInner}>
+                <img
+                    src={iconForKind(kind ?? "info")}
+                    alt={kind}
+                    className={css(styles.icon)}
+                />
+                <View style={styles.labelAndButtonsContainer}>
+                    <View style={styles.labelContainer}>
+                        <LabelSmall>{text}</LabelSmall>
+                    </View>
+                    {actions && (
+                        <View style={styles.actionsContainer}>
+                            {renderActions()}
+                        </View>
+                    )}
+                </View>
+                {onDismiss ? (
+                    <IconButton
```

Comment:
In a following PR, I will add an `aria-label` prop for the dismiss button and a default aria-label otherwise.